src/extension.ts

```diff
@@ -412,17 +390,21 @@ export async function activate(context: vscode.ExtensionContext) {
                         label: "ESP32S2BETA", target: "esp32s2beta" },
                 ],
                 { placeHolder: enterDeviceTargetMsg },
-            ).then((selected) => {
+            ).then(async (selected) => {
                 if (typeof selected === "undefined") {
                     return;
                 }
-                idfConf.writeParameter("idf.adapterTargetName", selected.target);
+                const configurationTarget = await idfConf.chooseConfigurationTarget();
+                idfConf.writeParameter("idf.adapterTargetName", selected.target, configurationTarget);
                 if (selected.target === "esp32") {
```

Comment:
Is it possible to support new targets without hardcoding them here? 
I.e. move to some config-file, or at least separate JS file.
(and we should add esp32s2, probably in separate PR)